package.json

```diff
@@ -52,6 +52,6 @@
 		"jest-environment-jsdom": "^29.7.0",
 		"jsdom": "^22.1.0",
 		"preact": "10.15.1",
-		"preact-render-to-string": "^6.2.2"
+		"preact-render-to-string": "^6.4.0"
```

Comment:
You'll need to bump this in `peerDependencies` too, as users won't be able to use this with the existing range of `>=5`

Reply:
Fixed it.
https://github.com/preactjs/preact-iso/pull/10/commits/ea1691968cea1bd90cca4909d701e9360652654b

Could this be a substantive breaking change?

Reply:
Eh, I'd say no.

Some people will need to upgrade, but that's not too big of an issue. I'll put it behind a new minor at least.